specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/searchindex.json

```diff
@@ -0,0 +1,2826 @@
+          },
+          {
+            "value": "sr-me",
+            "name": "SrMe",
+            "description": "Query language value for Serbian (Montenegro)."
+          },
+          {
+            "value": "sr-rs",
+            "name": "SrRs",
+            "description": "Query language value for Serbian (Serbia)."
+          },
+          {
+            "value": "sk-sk",
+            "name": "SkSk",
+            "description": "Query language value for Slovak (Slovakia)."
+          },
+          {
+            "value": "nb-no",
+            "name": "NbNo",
+            "description": "Query language value for Norwegian (Norway)."
+          },
+          {
+            "value": "hy-am",
+            "name": "HyAm",
+            "description": "Query language value for Armenian (Armenia)."
+          },
+          {
+            "value": "bn-in",
+            "name": "BnIn",
+            "description": "Query language value for Bengali (India)."
+          },
+          {
+            "value": "eu-es",
+            "name": "EuEs",
+            "description": "Query language value for Basque."
+          },
+          {
+            "value": "gl-es",
+            "name": "GlEs",
+            "description": "Query language value for Galician."
+          },
+          {
+            "value": "gu-in",
+            "name": "GuIn",
+            "description": "Query language value for Gujarati (India)."
+          },
+          {
+            "value": "he-il",
+            "name": "HeIl",
+            "description": "Query language value for Hebrew (Israel)."
+          },
+          {
+            "value": "ga-ie",
+            "name": "GaIe",
+            "description": "Query language value for Irish (Ireland)."
+          },
+          {
+            "value": "kn-in",
+            "name": "KnIn",
+            "description": "Query language value for Kannada (India)."
+          },
+          {
+            "value": "ml-in",
+            "name": "MlIn",
+            "description": "Query language value for Malayalam (India)."
+          },
+          {
+            "value": "mr-in",
+            "name": "MrIn",
+            "description": "Query language value for Marathi (India)."
+          },
+          {
+            "value": "fa-ae",
+            "name": "FaAe",
+            "description": "Query language value for Persian (U.A.E.)."
+          },
+          {
+            "value": "pa-in",
+            "name": "PaIn",
+            "description": "Query language value for Punjabi (India)."
+          },
+          {
+            "value": "te-in",
+            "name": "TeIn",
+            "description": "Query language value for Telugu (India)."
+          },
+          {
+            "value": "ur-pk",
+            "name": "UrPk",
+            "description": "Query language value for Urdu (Pakistan)."
+          }
+        ]
+      },
+      "description": "The language of the query."
```

Comment:
This parameter exists for backwards compatibility. In this API version, it's used for speller only, and speller is available in a subset of these languages.